test/boost/sstable_datafile_test.cc

```diff
@@ -6685,3 +6685,33 @@ SEASTAR_TEST_CASE(test_zero_estimated_partitions) {
         return make_ready_future<>();
     });
 }
+
+SEASTAR_TEST_CASE(test_cannot_have_partition_tombstones_metadata) {
```

Comment:
The test is a bit simplistic but better than nothing. An extension would be to have more than one partition and only one of them having a partition tombstone.

Reply:
done